RiotSwiftUI/Modules/Authentication/Common/AuthenticationModels.swift

```diff
@@ -86,6 +86,10 @@ class HomeserverAddress: NSObject {
     /// - Ensure the address contains a scheme, otherwise make it `https`.
     /// - Remove any trailing slashes.
     static func sanitized(_ address: String) -> String {
+        guard !address.isEmpty else {
+            // prevent prefixing an empty string with "https:"
+            return address
+        }
```

Comment:
I'm curious what triggered the need for this? The method isn't called until after the server is selected, and the button is disabled when it is empty. The address should never be able to become empty as far as I can tell.

Reply:
Totally right, I forgot about the legacy flow when looking at this. Makes sense 👍